### PR TITLE
 Implement Painter's Partition Problem (Binary Search) 

### PR DESCRIPTION
**Pull request by rk05-lenavo**
#599 Fixed Issue

### CODE REVIEW

File: Code/C++/ Implement_Painter's_Partition.cpp [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		best = min(best, max(partition(arr, i, k - 1),
+								sum(arr, i, n - 1)));
+
+	return best;
+}
+
+int main()
+{
+	int arr[] = { 10, 10, 10, 10};
+	int n = sizeof(arr) / sizeof(arr[0]);
```
Comment (rudrakshi99) [inline Code/C++/ Implement_Painter's_Partition.cpp:32]: make it user input

File: Code/C++/ Implement_Painter's_Partition.cpp [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#include <climits>
```
Comment (rudrakshi99) [inline Code/C++/ Implement_Painter's_Partition.cpp:1]: add a brief description of algo

File: Code/C++/ Implement_Painter's_Partition.cpp [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+	int arr[] = { 10, 10, 10, 10};
+	int n = sizeof(arr) / sizeof(arr[0]);
+	int k = 2;
+	cout << partition(arr, n, k) << endl;
+
+	return 0;
+}
+
+// Input:  k=2 , A={10, 10, 10, 10}
+// Output: 20
```
Comment (rudrakshi99) [inline Code/C++/ Implement_Painter's_Partition.cpp:40]: add atleast 2 test cases and complexities

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by rudrakshi99** [comment]
Please follow proper guidelines and also update the readme of sorting_searching folder.

**Comment by rk05-lenavo** [comment]
@rudrakshi99 mam is it okay now.

**Comment by rk05-lenavo** [comment]
@rudrakshi99 mam can you check

**Comment by rk05-lenavo** [comment]
@rudrakshi99 I have been requesting to you to see the code for so long.

**Comment by Rohan27s** [comment]
Plagiarism from https://www.geeksforgeeks.org/painters-partition-problem-set-2/ (you have just changed variable names)

**Pull request closed without merging**